### DISCUSSION TITLE
大家求求了帮我！！！

### DISCUSSION THREAD

**Discussion by MengXingCYF** [Meloong-Git/PCL#1860] [提问] [answered]
[错误报告-2023-05-14_21.04.11.zip](https://github.com/Hex-Dragon/PCL2/files/11472034/-2023-05-14_21.04.11.zip)

**Comment by Chiloven945** [comment]


**Reply by MengXingCYF** [reply]
怎么关闭

**Reply by WForst-Breeze** [reply]
> 怎么关闭

[见此](https://github.com/Hex-Dragon/PCL2/discussions/1740#discussioncomment-5787939)

**Reply by IF-desu** [reply]
> 怎么关闭

![747af3d08c58cb1b.png](https://github.com/Hex-Dragon/PCL2/assets/86994398/5cf6497e-7079-41c8-ad7e-f97f72d72e65)

**Reply by LTCatt** [reply]
UTF-8 导致的崩溃已经彻底修复了，不是这个原因导致的，不知道可以不说不要乱说……

**Comment by LTCatt** [comment, marked as answer]
如果使用 HMCL 启动没有问题可以提交 issue
